build/repo.targets

```diff
@@ -226,12 +223,6 @@
   </Target>
 
   <Target Name="ComputeGraph" DependsOnTargets="ResolveRepoInfo;GeneratePropsFiles">
```

Comment:
Technically this is more like "LoadSubmoduleGraph" now but it's probably not worth changing the name.